### PR TITLE
HCAP-1019 | Refactor participant table forms

### PR DESCRIPTION
**Pull request by arranfw**
https://freshworks.atlassian.net/browse/HCAP-1019

### CODE REVIEW

File: client/src/components/modal-forms/NewParticipantForm.js [hunk 1]
```diff
@@ -1,21 +1,60 @@
...
+        emailAddress: '',
+        origin: '',
+        otherOrigin: '',
+        hcapOpportunity: true,
+        contactedDate: '',
+        hiredDate: '',
+        startDate: '',
+        site: '',
+        acknowledge: false,
+      }}
```
Comment (dbayly-freshworks) [inline client/src/components/modal-forms/NewParticipantForm.js:52]: Nit maybe add initial values as a constant at the top of the file?

File: client/src/components/modal-forms/ArchiveHiredParticipantForm.js [hunk 2]
```diff
@@ -1,22 +1,34 @@
 import React from 'react';
 import { Box } from '@material-ui/core';
 import Grid from '@material-ui/core/Grid';
+import moment from 'moment';
```
Comment (dbayly-freshworks) [inline client/src/components/modal-forms/ArchiveHiredParticipantForm.js:4]: We use moment in a ton of projects, but it's no longer current. What are your thoughts of using a new date library in future projects? Do you have any favourites?

Comment (fwpushan) [reply client/src/components/modal-forms/ArchiveHiredParticipantForm.js:4]: I used dayjs .... that is lightweight.

**Review by fwpushan — DISMISSED** [review]
I already review this as part of previous draft PR

File: client/src/components/modal-forms/ArchiveHiredParticipantForm.js (hunk 2 above)
Comment (arranfw) [reply client/src/components/modal-forms/ArchiveHiredParticipantForm.js:4]: `date-fns` has been my replacement for moment

**Review by dbayly-freshworks — APPROVED** [review]

**Pull request merged**